addons/website_payment/models/res_config_settings.py

```diff
@@ -1,70 +1,21 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import api, fields, models, _
+from odoo import api, models
 
 
 class ResConfigSettings(models.TransientModel):
     _inherit = 'res.config.settings'
 
-    providers_state = fields.Selection(
-        selection=[
-            ('none', 'None'),
-            ('paypal_only', 'Paypal Only'),
-            ('other_than_paypal', 'Other than Paypal')
-        ],
-        compute='_compute_providers_state')
-    first_provider_label = fields.Char(
-        compute='_compute_providers_state')
-    is_stripe_supported_country = fields.Boolean(
-        related='company_id.country_id.is_stripe_supported_country')
-
     def _get_activated_providers(self):
-        self.ensure_one()
-        wire_transfer = self.env.ref('payment.payment_provider_transfer', raise_if_not_found=False)
-        return self.env['payment.provider'].search([
-            ('state', '!=', 'disabled'),
-            ('id', '!=', wire_transfer.id if wire_transfer else False),
-            '|',
-            ('website_id', '=', False),
-            ('website_id', '=', self.website_id.id)
-        ])
+        return super()._get_activated_providers().filtered_domain(
+            ['|', ('website_id', '=', False), ('website_id', '=', self.website_id.id)]
+        )
```

Comment:
Instead you create method `_get_avtivated_providers_domain` or something like that to avoid getting unwanted provider  in first place